Tests/Resources/breadcrumb.json

```diff
@@ -0,0 +1,3139 @@
+{
```

Comment:
Can we trim down this a little? Rather large file

Reply:
@bruno-garcia I just copied from other sample and not sure what to trim, `fatalError` resource. sentry event itself has empty extra, tags and the breadcrumb, all the other things are about SentryCrash

Reply:
I'd remove anything that is not related to the `breadcrumb` (assuming based on the name) test case. The biggest offender there is the loaded images though

Reply:
`binary_images`

Reply:
yep it seems to be working, my 1st try failed on `isValidReport` and I just copied a sample, I was probably guessing it was a normal sentry event :D good catch.